### PR TITLE
doc-api-update-and-delete-post

### PR DESCRIPTION
**Pull request by at-trangle**
## Description
doc-api-update-and-delete-post
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: docs/api-references/post.md [hunk 1]
```diff
@@ -85,6 +85,35 @@ Get list all product's posts
     "code": 200
 }
 ```
+### `POST` Update post
```
Comment (at-phuongvu) [inline docs/api-references/post.md]: Method `Put`

File: docs/api-references/post.md [hunk 2]
```diff
@@ -85,6 +85,35 @@ Get list all product's posts
...
+#### Request body
+
+| Key | Type | Description |
+|---|---|---|
+| post | int | required |
+| type | int | required (1) |
+| content | String | required if rating is null |
+| rating | int | required if content is null |
+
+#### Response
```
Comment (at-phuongvu) [inline docs/api-references/post.md]: Sửa lại response đúng form nha

File: docs/api-references/post.md [hunk 3]
```diff
@@ -85,6 +85,35 @@ Get list all product's posts
     "code": 200
 }
 ```
+### `POST` Update post
+```
+/api/posts/{post}/update
```
Comment (at-phuongvu) [inline docs/api-references/post.md]: link api không có chữ `update`

File: docs/api-references/post.md [hunk 4]
```diff
@@ -85,6 +85,140 @@ Get list all product's posts
...
+            "user_id": 1,
+            "type": 1,
+            "content": "Voluptate doloremque rerum dignissimos dolores rerum. Blanditiis et qui sit ea nobis rem. Qui ullam aut aut a fugit aut. Magni voluptatem et ut aut. Sint iusto error quisquam deserunt sit est doloribus magnam. Tempora aliquam optio a. Et illum sapiente omnis sequi consequatur molestiae accusantium distinctio. Aut commodi asperiores perspiciatis ut. Ad sequi velit incidunt tenetur.",
+            "rating": 4,
+            "status": 1,
+            "diff_time": "1 day ago",
+            "image_path": "http://product-tiki.show/images/upload/",
+            "created_at": "2018-05-31 07:04:58",
+            "updated_at": "2018-05-31 07:04:58",
+            "deleted_at": null,
```
Comment (at-quangtran) [inline docs/api-references/post.md]: delete rồi mà return trường này null?

**Pull request merged**